### PR TITLE
chore(docs): Text Area docs update (VIV-2110)

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

File: libs/components/src/lib/text-area/README.md [hunk 1]
```diff
@@ -1,199 +1,138 @@
...
-- Type: `string` | `undefined`
-- Default: `undefined`
-
-```html preview
-<vwc-text-area
-	label="Success text below"
-	success-text="Success text"
-></vwc-text-area>
+<your-prefix-text-area
+	label="First name"
```
Comment (TaylorJ76) [inline libs/components/src/lib/text-area/README.md:28]:
```suggestion
	label="Description"
```

File: libs/components/src/lib/text-area/README.md [hunk 2]
```diff
@@ -1,199 +1,138 @@
...
+		inline-size: 240px;
+	}
+</style>
 ```
 
-### Readonly
+## Slots
 
-Add the `readonly` attribute to restrict user from changing the text area's value.
+### Helper-Text
```
Comment (TaylorJ76) [inline libs/components/src/lib/text-area/README.md:87]:
```suggestion
### Helper Text Slot
```

File: libs/components/src/lib/text-area/README.md [hunk 3]
```diff
@@ -1,199 +1,138 @@
...
-## Slots
+### Slots
 
-### Helper-Text
+<div class="table-wrapper">
 
-The `helper-text` slot allows you to use rich content as the text area's helper text.
+| Name            | Description                         |
+| --------------- | ----------------------------------- |
+| **helper-text** | Sets the input element's appearance |
```
Comment (TaylorJ76) [inline libs/components/src/lib/text-area/README.md:131]:
```suggestion
| `helper-text` | Sets the input element's appearance |
```

File: libs/components/src/lib/text-area/README.md [hunk 4]
```diff
@@ -1,199 +1,138 @@
...
 
-### Readonly
+## Slots
 
-Add the `readonly` attribute to restrict user from changing the text area's value.
+### Helper-Text
 
-- Type: `boolean`
-- Default: `false`
+The `helper-text` slot allows you to use rich content as the text area's helper text.
```
Comment (TaylorJ76) [inline libs/components/src/lib/text-area/README.md:89]:
```suggestion
The `helper-text` slot allows you to use rich content as the Text Area's helper text.
```

File: libs/components/src/lib/text-area/VARIATIONS.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```html preview
+<vwc-text-area
+	label="Success text below"
+	success-text="Success text"
+></vwc-text-area>
+```
+
+## Rows
+
+Use the `rows` attribute to set the number of visible rows of text in the text area.
```
Comment (TaylorJ76) [inline libs/components/src/lib/text-area/VARIATIONS.md:91]:
```suggestion
Use the `rows` attribute to set the number of visible rows of text in the Text Area.
```

**Review by TaylorJ76 — CHANGES_REQUESTED** [review]

File: libs/components/src/lib/text-area/README.md (hunk 3 above)
Comment (rachelbt) [reply libs/components/src/lib/text-area/README.md:131]: but this is the way alert, card, checkbox, buttons and more are.
https://vivid.deno.dev/components/alert/code/#slots-1
https://vivid.deno.dev/components/button/code/#slots-1

Comment (TaylorJ76) [reply libs/components/src/lib/text-area/README.md:131]: Fair enough. Can we change the ones in the Properties and Event tables to match then please?

**Review by TaylorJ76 — APPROVED** [review]

**Pull request merged**